bears/js/HappinessLintBear.py

```diff
@@ -19,6 +19,19 @@ class HappinessLintBear:
     ASCIINEMA_URL = 'https://asciinema.org/a/80714'
     CAN_DETECT = {'Syntax'}
 
+    def __init__(self, use_spaces, queue):
+        self.use_spaces = use_spaces
+        self.queue = queue
+
     @staticmethod
-    def create_arguments(filename, file, config_file):
+    def create_arguments(
+            filename, file, config_file,
+            use_spaces: bool = False):
+
+        if (self.use_spaces):
```

Comment:
dont use `self`

Reply:
@newbazz you did not complete the changes here

Reply:
I did but while experimenting a bit i forgot to undo it so ;P, I have done it now.